stargate-redis/test/integration/McSsoControllerSpec.scala

```diff
@@ -17,16 +15,17 @@ import play.api.Configuration
 import play.api.libs.json.{JsDefined, JsString}
 import play.api.mvc.{ControllerComponents, Session}
 import play.api.http.SessionConfiguration
-
 import com.salesforce.mce.stargate.controllers.McSsoController
-import com.salesforce.mce.stargate.models.McSsoDecodedJwt
+import com.salesforce.mce.stargate.models.{McSsoDecodedJwt}
 import com.salesforce.mce.stargate.services.SessionTrackingService
 import com.salesforce.mce.stargate.services.impl.SessionTrackingServiceRedisImpl
 import com.salesforce.mce.stargate.utils.{JedisConnection, JwtUtil, JwtUtilImpl}
 import play.api.http.{SecretConfiguration, SessionConfiguration}
 import play.api.libs.crypto.{CSRFTokenSignerProvider, DefaultCookieSigner}
 import play.filters.csrf.{CSRFAddToken, CSRFConfig}
 
+import scala.collection.immutable.Map
```

Comment:
let's organize the import with the following order:

```
import java.*
import javax.*
__blankline__
import scala.*
__blankline__
import play.*
__blankline__
import any.other.third.party.dependencies.*
__blankline__
import com.salesforce.mce.*
```

Reply:
Updated in latest commit